### DISCUSSION TITLE
换头像

### DISCUSSION THREAD

**Discussion by yihangyingzi** [share121/inter-knot#3135] [其他]
怎么换头像啊？

**Comment by alannnnnn22** [comment]
这个委托我接了！

**Comment by Abandon12393** [comment]
左上角，进入github，再点击头像，头像左下角有一个edit，点进去就能换

**Reply by 15666851105** [reply]
不知道为啥，我换一个图片儿他就说不行。(╥_╥)

**Comment by HyperMuteki0347** [comment]
多试几次就行了不会很麻烦

**Reply by 15666851105** [reply]
可我已经试了n遍了(／_＼)大怨种

**Reply by HyperMuteki0347** [reply]
我刚刚还换了一次头像。总之多试几次，或者换张图试试。

**Comment by 15666851105** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >        
好的

**Reply by HyperMuteki0347** [reply]
这个米游社表情我在手机端弄了半天没弄好，等会用电脑试试

**Reply by 15666851105** [reply]
![thumb1720849392424](https://github.com/user-attachments/assets/b717be21-d6d4-442a-9b50-100d8efebea6)
这个要用油猴，有的手机浏览器自带油猴，我用的是via

**Reply by 15666851105** [reply]
到时候你点那个帖子，点他那个安装脚本的那个，一遍不行，多安几遍，最后再在评论看看，得等几秒它就自动从下面弹出来

**Comment by Nie7bai** [comment]
头向只能是rpg、gif、（还有一个忘了），还有不能超过1MB<img src="https://img-static.mihoyo.com/communityweb/upload/0b5acb4c4e49bf8733806b11450f6c50.png" alt="三月七_Biu" width="75" >

**Reply by 15666851105** [reply]
好的，谢谢<img src="https://bbs-static.miyoushe.com/static/2024/06/19/52971140330611d82cad599f20788ab6_3848320906174398412.png" alt="帕姆 开拓" width="75" >
